examples/training/object_detection/pascal_voc/retina_net.py

```diff
@@ -363,31 +170,33 @@ def pad_fn(image, boxes):
         # (0, 1).  In our case, we feed our model images with inputs in the range (0, 255).
         include_rescaling=True,
     )
-    # Fine-tuning a RetinaNet is as simple as setting backbone.trainable = False
-    model.backbone.trainable = False
+# Fine-tuning a RetinaNet is as simple as setting backbone.trainable = False
+model.backbone.trainable = False
+optimizer = tf.optimizers.SGD(learning_rate=BASE_LR, global_clipnorm=10.0)
 
-    optimizer = tf.optimizers.SGD(learning_rate=BASE_LR, global_clipnorm=10.0)
 model.compile(
     classification_loss="focal",
     box_loss="smoothl1",
     optimizer=optimizer,
+    metrics=[
+        keras_cv.metrics.COCORecall(bounding_box_format="xywh", class_ids=range(20)),
+        keras_cv.metrics.COCOMeanAveragePrecision(
+            bounding_box_format="xywh", class_ids=range(20)
+        ),
+    ],
 )
 
 
-def convert_to_dict(images, boxes):
-    return images, {"gt_boxes": boxes[:, :, :4], "gt_classes": boxes[:, :, 4]}
-
-
 callbacks = [
     keras.callbacks.TensorBoard(log_dir="logs"),
     keras.callbacks.ReduceLROnPlateau(patience=5),
     keras.callbacks.EarlyStopping(patience=10),
     keras.callbacks.ModelCheckpoint(CHECKPOINT_PATH, save_weights_only=True),
-    PyCOCOCallback(eval_ds.map(convert_to_dict), "xywh"),
 ]
 
 history = model.fit(
     train_ds,
-    epochs=100,
+    validation_data=eval_ds.take(20),
```

Comment:
Is the call to `.take` here intentional? I'd think we would want the whole eval DS?